src/main.js

```diff
@@ -122,7 +122,7 @@ export default class Auth {
 			let data = await response.json();
 
 			// If the response returned an error, try to get a Firebase error code/message.
-			// Sometimes the error codes are joined with an explanation, we don't need that(its a bug).
+			// Sometimes the error codes are joined with an explanation, we don't need that (its a bug).
```

Comment:
I think, "it's a bug" should be fixed too

Reply:
Agreed.